stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -137,24 +144,21 @@ public fun MessageComposer(
             }
 
             Row(
-                Modifier
-                    .fillMaxWidth(),
-                verticalAlignment = CenterVertically
+                Modifier.fillMaxWidth(),
+                verticalAlignment = Bottom
             ) {
 
                 if (shouldShowIntegrations && activeAction !is Edit) {
                     integrations()
                 } else {
                     Spacer(
-                        modifier = Modifier
-                            .size(48.dp)
-                            .align(CenterVertically)
+                        modifier = Modifier.size(16.dp)
                     )
                 }
 
                 input(messageInputState)
 
```

Comment:
Just disabling the "send" button for now. Will show a toast or a snackbar in the next PR as like with suggestion popup it is hard to display it above the composer.

Reply:
Is it possible to show some kind of errors within either the attachments (as we discussed) or the composer? Dedicate a small slot?

We can create this as an extra ticket, for UI/UX improvement

Reply:
Yes, I will create a separate ticket for validation errors. Needs to be checked with the design team.

Reply:
https://github.com/GetStream/stream-chat-android/issues/2653